client/a8c-for-agencies/sections/migrations/primary/migrations-overview-v2/migrate-site-button.tsx

```diff
@@ -0,0 +1,125 @@
+import { Gridicon, WordPressLogo, Popover } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import { CONTACT_URL_FOR_MIGRATION_OFFER_HASH_FRAGMENT } from 'calypso/a8c-for-agencies/components/a4a-contact-support-widget';
+import A4ALogo from 'calypso/a8c-for-agencies/components/a4a-logo';
+import PopoverMenuItem from 'calypso/a8c-for-agencies/components/a4a-popover/menu-item';
+import {
+	A4A_MIGRATIONS_MIGRATE_TO_WPCOM_LINK,
+	A4A_MIGRATIONS_MIGRATE_TO_PRESSABLE_LINK,
+} from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import pressableIcon from 'calypso/assets/images/pressable/pressable-icon.svg';
+import preventWidows from 'calypso/lib/post-normalizer/rule-prevent-widows';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+const MigrateSiteButton = () => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const popoverMenuContext = useRef( null );
+
+	const [ isMenuVisible, setMenuVisible ] = useState( false );
+
+	const toggleMenu = () => {
+		setMenuVisible( ( isVisible ) => ! isVisible );
+	};
+
+	const popoverContent = (
```

Comment:
Same as my previous comment on another PR. We can wrap this with `useMemo` to avoid redrawing it or just move it within the template.

Reply:
Fixed it here: https://github.com/Automattic/wp-calypso/pull/96401/commits/8af12d3f38c8042fa4aa2c8f60c0ec64c4f2e64e.